common/src/main/java/org/jboss/da/common/util/Configuration.java

```diff
@@ -18,7 +24,25 @@ public DAConfig getConfig() throws ConfigurationParseException {
 
         try (InputStream configStream = getConfigStream()) {
             ObjectMapper mapper = new ObjectMapper();
-            return mapper.readValue(configStream, DAConfig.class);
+
+            // Note: I need to add the dummy classes GlobalGroupWrapper and
+            // PNCGroupWrapper since the version of Jackson we are using
+            // doesn't support FAIL_ON_INVALID_SUBTYPE (it's in v2.2)
+            mapper.registerSubtypes(new NamedType(GlobalGroupWrapper.class, "global"));
+            mapper.registerSubtypes(new NamedType(PNCGroupWrapper.class, "pnc"));
```

Comment:
Did you tried to use Default implementation? Or codehause jackon doesn't support it?